draft-ietf-tls-dtls13.md

```diff
@@ -471,7 +479,7 @@ unpacked RecordNumber structure, as shown below:
 ~~~
 %%% Record Layer
     struct {
-        uint16 epoch;
+        uint64 epoch;
         uint48 sequence_number;
     } RecordNumber;
 ~~~
```

Comment:
Down on line 479, it says "This 64-bit value is used in the ACK message as well as in the "record_sequence_number" input to the AEAD function.

Two problems:

1. It's no longer 64 bits.  It's 112 bits.
2. It's no longer 64 bits.  It's 112 bits.

The first is just correcting the text.

The second is that the per-record nonce construction in https://datatracker.ietf.org/doc/html/rfc8446#section-5.3 no longer works for most, if not all, modern ciphers without additional transforms (which would reduce, not improve security).

I think that the fix here is to make the epoch 48 bits rather than 64.  Still, this will need a lot more text in order to make the generation of the per-record nonce clear.  You need to point to Section 5.3 of 8446 and say what is different and why.

(I searched for "record_sequence_number" in this document and RFC 8446 and found no other mentions of the string.  I'm sure that people without context will really struggle with this.)